### PR TITLE
Re-activate and Update Ubuntu.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/Ubuntu.xml [hunk 1]
```diff
@@ -1,137 +1,80 @@
...
-<ruleset name="Ubuntu.com (partial)" default_off='failed ruleset test'>
-
-	<!--	Direct rewrites:
-					-->
+<ruleset name="Ubuntu.com (partial)">
 	<target host="ubuntu.com" />
+	<target host="www.ubuntu.com" />
 	<target host="apps.ubuntu.com" />
 	<target host="assets.ubuntu.com" />
+		<test url="http://assets.ubuntu.com/v1/0df231d4-core-print.css" />
```
Comment (J0WI) [inline src/chrome/content/rules/Ubuntu.xml]: buy.ubuntu.com
certification.ubuntu.com

File: src/chrome/content/rules/Ubuntu.xml [hunk 2]
```diff
@@ -1,137 +1,80 @@
...
 	<target host="errors.ubuntu.com" />
+	<target host="fridge.ubuntu.com" />
 	<target host="help.ubuntu.com" />
 	<target host="insights.ubuntu.com" />
-	<target host="lists.ubuntu.com" />
+	<target host="irclogs.ubuntu.com" />
+	<target host="juju.ubuntu.com" />
 	<target host="maas.ubuntu.com" />
-	<target host="one.ubuntu.com" />
+	<target host="packages.ubuntu.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Ubuntu.xml]: partners.ubuntu.com

File: src/chrome/content/rules/Ubuntu.xml [hunk 3]
```diff
@@ -1,137 +1,80 @@
...
 	<target host="developer.ubuntu.com" />
 	<target host="myapps.developer.ubuntu.com" />
+	<target host="entropy.ubuntu.com" />
 	<target host="errors.ubuntu.com" />
+	<target host="fridge.ubuntu.com" />
 	<target host="help.ubuntu.com" />
 	<target host="insights.ubuntu.com" />
-	<target host="lists.ubuntu.com" />
+	<target host="irclogs.ubuntu.com" />
+	<target host="juju.ubuntu.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Ubuntu.xml]: merges.ubuntu.com

File: src/chrome/content/rules/Ubuntu.xml [hunk 4]
```diff
@@ -1,137 +1,80 @@
...
 	<target host="maas.ubuntu.com" />
-	<target host="one.ubuntu.com" />
+	<target host="packages.ubuntu.com" />
 	<target host="paste.ubuntu.com" />
+	<target host="pastebin.ubuntu.com" />
 	<target host="patches.ubuntu.com" />
 	<target host="jenkins.qa.ubuntu.com" />
-	<target host="uds.ubuntu.com" />
+	<target host="shop.ubuntu.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Ubuntu.xml]: tutorials.ubuntu.com

File: src/chrome/content/rules/Ubuntu.xml [hunk 5]
```diff
@@ -1,137 +1,80 @@
...
-	ᵘ Unsecurable, doesn't trip MCB
-	
-	Invalid certificate break redirects by server:
-		cloud
-		juju
-		
+		+ Ubuntu-CN.xml
+		+ Ubuntu_Forum.xml
+
+	Non-functional hosts
```
Comment (J0WI) [inline src/chrome/content/rules/Ubuntu.xml]: ask.ubuntu.com
bugs.ubuntu.com
jobs.ubuntu.com
tour.ubuntu.com

**Comment by cschanaj** [comment]
@J0WI fixed. thanks!

**Pull request merged**